dftimewolf/lib/processors/turbinia.py

```diff
@@ -102,26 +102,51 @@ def process(self):
     request = TurbiniaRequest()
     request.evidence.append(evidence_)
 
+    # Get threat intelligence data from any modules that have stored some.
+    observables = self.state.get_data('threat_intelligence')
+    if observables:
```

Comment:
As discussed, adding a comment here as to what "observables" is supposed to look like would be nice. Perhaps also make an Observable class to make this more checkable

Reply:
Done.